bokehjs/src/coffee/api/typings/models/plots.d.ts

```diff
@@ -3,6 +3,11 @@ declare namespace Bokeh {
         plot?: Plot;
     }
 
+    export interface PlotView extends View<Plot> {
+        resize_width_height(use_width: boolean, use_height: boolean, maintain_ar?: boolean,
+                            width?: number, height?: number): void;
+    }
+
```

Comment:
Ah, thanks for the tip. The view was set to `any` anyways (until recently), but I thought I'd add some typing already so we wont forget this function.